cmd/testrunner.go

```diff
@@ -189,9 +189,7 @@ func testTypeCommandActionFactory(runner testrunner.TestRunner) cobraext.Command
 			return fmt.Errorf("cannot determine if package has data streams: %w", err)
 		}
 
-		// Temporarily until independent Elastic Agents are enabled by default,
-		// enable independent Elastic Agents if package defines that requires root privileges
-		runIndependentElasticAgent := manifest.Agent.Privileges.Root
+		runIndependentElasticAgent := false
```

Comment:
Here, in `cmd` just check about the environment variable.
Checks about `agent.privileges.root` in package or data-stream manifest have been moved to `prepareScenario` method in system test runner.